docs/pages/generated/client/bindings.rst

```diff
@@ -321,3 +321,59 @@ let
 
 
 
+Knockout Binding Defaults
+-------------------------
+
+These are static properties you can assign once somewhere in the app lifecycle startup to change the default markup generated server-side for the Knockout bindings.  Currently, there are defaults for the Bootstrap grid system width of ``<label>`` and ``<intput>`` tags, as well as default formats for the date pickers.
```

Comment:
Suggest adding a little bit of context here, since we don't actually have documentation for the html helpers at all: 

```suggestion
These are static properties on :csharp:`IntelliTect.Coalesce.Knockout.Helpers.Knockout` you can assign to somewhere in the app lifecycle startup to change the default markup generated server-side when using ``@Knockout.*`` methods to render Knockout bindings in your ``.cshtml`` files.  Currently, there are defaults for the Bootstrap grid system width of ``<label>`` and ``<input>`` tags, as well as default formats for the date pickers.
```

Reply:
Good idea